### ISSUE TITLE
Add unique and descriptive labels for question buttons

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-core#656]
### Subject of the issue
The following issue has been reported by an external accessibility auditor:

> Identical label “Show correct answer”, "Show your answer", "Show feedback" and "Submit" are specified for the buttons.
> As a result, screen reader users might not understand the purpose of the buttons effectively.

![Image](https://github.com/user-attachments/assets/c80eb9a4-04f3-4652-9758-41114752ecb4)

### Steps to reproduce
1. Using screen reader, navigate to the mentioned buttons.
2. Listen that the buttons label are identical.

### Expected behaviour
The screen reader should announce unique and descriptive labels for the buttons. 

Recommendation provided by the auditor:

> Provide unique and descriptive labels for the buttons using `aria-label` attribute.
> 
> Alternatively apply the following changes:
> 
> · Remove `aria-label` attribute from the source code of mentioned buttons.
> 
> · Specify `aria-labelledby` attribute to `<button>` element containing the mentioned buttons.
> 
> · Specify `id` attribute with unique value to `<button>` element containing mentioned buttons such as "Show feedback" and `<div>` element containing the heading text such as "Multiple Choice Question".
> 
> · Reference both `id` attribute values of `<button>` and `<div>` elements to `aria-labelledby` attribute respectively.

Based on the above solution, I'd suggest either adding the component `title` to the button `aria-label`. Or, use `aria-labelledby` as suggested to reference the component `title`.

**Comment by oliverfoster** [comment]
Perhaps you could use a similar technique to the one used by mcq to group the items, but use it to group the buttons instead?

```jsx
<div 
  role="group"
  aria-labelledby={ariaQuestion ? null : (displayTitle || body || instruction) && `${_id}-header`}
  aria-label={ariaQuestion || null}
>
  <div>buttons stuff</div>
</div>
```

https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA17
